pkg/apis/planetscale/v2/vitessbackupstorage_types.go

```diff
@@ -67,6 +67,8 @@ type VitessBackupLocation struct {
 	GCS *GCSBackupLocation `json:"gcs,omitempty"`
 	// S3 specifies a backup location in Amazon S3.
 	S3 *S3BackupLocation `json:"s3,omitempty"`
+	// Azblob specifies a backup location in Azure Blob Storage.
+	Azblob *AzblobBackupLocation `json:"azblob,omitempty"`
```

Comment:
Nit - Probably worth spelling it out, even though Azblob sounds great.
```suggestion
	AzureBlob *AzureBlobBackupLocation `json:"azureblob,omitempty"`
```

Reply:
The API in golang is azblob, so thats what I called it in vitess and I feel the name should be the same here. It would be like saying SimpleStorageService instead of S3, if everyone knows it as azblob we should just use that name